static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -335,3 +335,8 @@ const StyledAvatarList = styled(AvatarList)`
 const TitleNumber = styled('span')`
   font-weight: normal;
 `;
+
+// Using 22px + space(1) = space(4)
+const SmallerSidebarWrap = styled(SidebarSection.Wrap)`
+  margin-bottom: 22px;
```

Comment:
only doing this because space(3) + space(1) != space(4). for some reason sapce(3) is 20px